src/galileo/config.py

```diff
@@ -0,0 +1,25 @@
+# mypy: disable-error-code=syntax
+# We need to ignore syntax errors until https://github.com/python/mypy/issues/17535 is resolved.
+from typing import Any, Optional
+
+from galileo_core.schemas.base_config import GalileoConfig as BaseConfig
+
+
+class GalileoConfig(BaseConfig):
```

Comment:
I'd strongly recommend renaming this to avoid confusion.
```suggestion
class GalileoPythonConfig(BaseConfig):
```